diff --git a/node_modules/movi-player/.bun-tag-210cf6c7a6827e58 b/.bun-tag-210cf6c7a6827e58
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/element.js b/dist/element.js
index 3669b6e669fc7b89a14f4b0cddb7da7796be88b8..e03111a86f376617af2cac2f920f99ef8396a7c4 100644
--- a/dist/element.js
+++ b/dist/element.js
@@ -110661,7 +110661,7 @@ class MoviElement extends HTMLElement {
         case "End":
           A2.preventDefault(), this.currentTime = this.duration, this.showControls();
       }
-    }), this.hasAttribute("tabindex") || this.setAttribute("tabindex", "0"), this.addEventListener("touchstart", () => {
+    }), this.addEventListener("touchstart", () => {
       this.lastTouchTime = Date.now();
     }, { passive: true, capture: true }), this.addEventListener("mouseenter", () => {
       Date.now() - this.lastTouchTime < 800 || this._controls && this.showControls();
diff --git a/dist/index.js b/dist/index.js
index 3669b6e669fc7b89a14f4b0cddb7da7796be88b8..e03111a86f376617af2cac2f920f99ef8396a7c4 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -110661,7 +110661,7 @@ class MoviElement extends HTMLElement {
         case "End":
           A2.preventDefault(), this.currentTime = this.duration, this.showControls();
       }
-    }), this.hasAttribute("tabindex") || this.setAttribute("tabindex", "0"), this.addEventListener("touchstart", () => {
+    }), this.addEventListener("touchstart", () => {
       this.lastTouchTime = Date.now();
     }, { passive: true, capture: true }), this.addEventListener("mouseenter", () => {
       Date.now() - this.lastTouchTime < 800 || this._controls && this.showControls();
